src/core/Surface/Surface.tsx

```diff
@@ -12,6 +12,8 @@ import '@itwin/itwinui-css/css/surface.css';
  */
 const getSurfaceElevationValue = (elevation: SurfaceProps['elevation']) => {
   switch (elevation) {
+    case 0:
+      return '0';
```

Comment:
Not sure..Docs say initial value is `none`
https://developer.mozilla.org/en-US/docs/web/css/box-shadow

Reply:
value changed to `none`